internal/nexodus/nexodus.go

```diff
@@ -566,6 +586,11 @@ func (nx *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		for _, proxy := range nx.proxies {
 			proxy.Start(ctx, wg, nx.userspaceNet)
 		}
+		if nx.exitNode.exitNodeClientEnabled {
+			if err := nx.ExitNodeClientSetup(); err != nil {
+				nx.logger.Errorf("failed to enable this device as an exit-node client: %v", err)
+			}
+		}
```

Comment:
This functionality needs to be integrated into the reconcile loop instead of just once at startup or when triggered via `nexctl`.

Reply:
Agree, tracked in #1408 ty ty.